api/src/pcapi/core/finance/api.py

```diff
@@ -468,6 +468,7 @@ def _mark_as_processed(pricings):
         .join(models.Pricing.booking)
         .join(offers_models.Stock)
         .join(models.Pricing.businessUnit)
+        .join(BankInformation)
```

Comment:
(détail / question de goût ?) Ta version marche bien, mais perso je préfère spécifier le _champ_ utilisé pour la jointure (et pas juste le modèle) : c'est plus explicite, moins magique (et ça évite peut-être aussi que SQLAlchemy se trompe, même si c'est peut-être impossible).

```suggestion
        .join(models.BusinessUnit.bankAccount)
```

(On notera que je n'ai pas fait ça pour la jointure de `offers_models.Stock`, au lieu de `bookings_models.Booking.stock`. Un oubli ! :) )

Reply:
Bien vu, j'en profite pour ajouter la précision sur la jointure de Stock.